pytorch_pipeline/examples/cifar10/utils.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+import json
+import os
+
+from sklearn.metrics import confusion_matrix
+from io import StringIO
+import boto3
+
+
+class Visualization:
+    def __init__(self):
+        self.parser_args = None
+
+    def _generate_confusion_matrix_metadata(self, confusion_matrix_path, vocab):
+        print("Generating Confusion matrix Metadata")
+        metadata = {
+            "type": "confusion_matrix",
+            "format": "csv",
+            "schema": [
+                {"name": "target", "type": "CATEGORY"},
+                {"name": "predicted", "type": "CATEGORY"},
+                {"name": "count", "type": "NUMBER"},
+            ],
+            "source": confusion_matrix_path,
+            # Convert vocab to string because for bealean values we want "True|False" to match csv data.
+            "labels": list(map(str, vocab)),
+        }
+        self._write_ui_metadata(
+            metadata_filepath="/mlpipeline-ui-metadata.json", metadata_dict=metadata
```

Comment:
The paths for outputs should not be hardcoded, but rather received from `{outputPath: MLPipeline UI Metadata}` placeholder.

Reply:
> The paths for outputs should not be hardcoded, but rather received from `{outputPath: MLPipeline UI Metadata}` placeholder in `component.yaml` and passed all the way through the program.

We are working on the visualization component. I will address this comment in a separate PR.